src/auto-evo/RunResults.cs

```diff
@@ -169,12 +169,19 @@ string PatchString(Patch patch)
 
             void OutputPopulationForPatch(Species species, Patch patch, long population)
             {
-                builder.Append("  ");
-
-                builder.Append(PatchString(patch));
-
-                builder.Append(" population: ");
-                builder.Append(Math.Max(population, 0));
+                if (population > 0)
+                {
+                    builder.Append("  ");
+                    builder.Append(PatchString(patch));
+                    builder.Append(" population: ");
+                    builder.Append(Math.Max(population, 0));
```

Comment:
`Math.Max` is no longer needed here as `population > 0` implies that the population is already positive.